wowup-electron/src/app/services/warcraft/warcraft-installation.service.ts

```diff
@@ -347,6 +357,17 @@ export class WarcraftInstallationService {
     return path.join(location, clientFolderName, executableName);
   }
 
+  private getFullLutrisProductPath(location: string, agentPath: string, clientType: WowClientType): string {
```

Comment:
See above linux comment.

Adding a platform specific step to "getInstalledProducts", after 'decodedProducts' is called might be a decent way to move this?
(IE: this._impl.resolveProducts(decodedProducts) or something)